libs/contract-core/package.json

```diff
@@ -48,13 +48,21 @@
   ],
   "main": "./build/index.js",
   "exports": {
+    "./contract": "./build/cjs/index.js",
```

Comment:
very shonky work-around lets us reference production contract in tests.

Reply:
(cause we can't validate code that connects to ext servers without it)

Reply:
Can this be used for e2e tests?

Reply:
Not quite the same: even E2E tests would normally only run under devlive; so no external connections.  Here I just wanted to be able to explicitly test the external code.

However, obvs your also right (and I haven't thought of this) that our E2E tests need to run under devlive, which is a very different environment to test.

Some thinking required...